misc/drone-scripts/setup.sh

```diff
@@ -33,8 +33,20 @@ do
     sleep 1;
 done
 
-echo "Reset complete"
-sleep 5;
+echo "Reset complete\nWaiting for VM to get IP..."
+
+until govc vm.ip photon.vmfs
```

Comment:
Yeah that's correct. I will work on the retry mechanism with timeout and generate the diff again.

Thanks!